clifford/__init__.py

```diff
@@ -301,12 +301,7 @@ def get_leftLaInv(mult_table, gradeList):
 
     @numba.njit
     def leftLaInvJIT(value):
-        intermed = np.zeros((n_dims, n_dims))
-        for test_ind, i in enumerate(k_list):
-            j = l_list[test_ind]
-            k = m_list[test_ind]
-            intermed[i, j] += mult_table_vals[test_ind] * value[k]
-        intermed = np.transpose(intermed)
```

Comment:
Compare to:

https://github.com/pygae/clifford/blob/93906afdd01594b1a1877ea3cf523dd936d3a3a0/clifford/__init__.py#L489-L498